cmd/indexer/commands/sequenial.go

```diff
@@ -9,7 +9,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
-func doSequential(cmd *cobra.Command, p *processor.Sequential) {
+func doSequential(cmd *cobra.Command, p *processor.Sequential, numRetry int) {
+	failureCounter := 0
```

Comment:
setting the counter here means that 10 errors at any time will cause it to fail. For a long running task like this reseting after each success should balance out any bumps in down stream resources with not getting stuck endlessly.

Reply:
Even I did the changes you mention, it was stucking in the loop. I use `backoff.WithMaxRetries(backoff.NewExponentialBackOff(), maxRetry)` to handle the repeating errors from a same block. When it reaches the limit, it will exit the loop.